sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/SimplifyConditionalsInPredicateSuite.scala

```diff
@@ -108,15 +108,18 @@ class SimplifyConditionalsInPredicateSuite extends PlanTest {
     Seq(Some(FalseLiteral), None, Some(Literal(null, BooleanType))).foreach { elseExp =>
       val originalCond = CaseWhen(
         Seq((UnresolvedAttribute("i") > Literal(10), UnresolvedAttribute("b"))),
-        elseExp)
+        elseExp).as("c")
```

Comment:
Add alias to avoid test failure:
```
== FAIL: Plans do not match ===
!Project [CASE WHEN (i#0 > 10) THEN b#0 END AS CASE WHEN (i > 10) THEN b ELSE NULL END#0]   Project [CASE WHEN (i#0 > 10) THEN b#0 END AS CASE WHEN (i > 10) THEN b END#0]
 +- LocalRelation <empty>, [i#0, b#0, a#0, m#0]                                             +- LocalRelation <empty>, [i#0, b#0, a#0, m#0]

```